sql/updates/world/6.x/2016_05_13_00_world.sql

```diff
@@ -0,0 +1,736 @@
+-- Tol Barad --
+
+-- Spawns are located half way down
+SET @CGUID := 376000; -- 214 creature entries (380000->)
+SET @OGUID := 68000; -- 111 gameobject entries
```

Comment:
Note to merger, remember to check for unused guids on 250000 for creatures and 200000 for gameobjects.